test/unit/calculators/minimum_wage_calculator_test.rb

```diff
@@ -161,6 +161,12 @@ class MinimumWageCalculatorTest < ActiveSupport::TestCase
           assert !@calculator.eligible_for_living_wage?
         end
       end
+
+      should 'return false if age is over 25, and date is before 2016-04-01' do
```

Comment:
Minor: The narrative here I feel should read thus:
```ruby should  'return false if age is over 25, and date is on or before 2016-04-01' do```

This is based on [line](https://github.com/alphagov/smart-answers/blob/national-minimum-wage-update/lib/smart_answer/calculators/minimum_wage_calculator.rb#L174) 174 in `lib/smart_answer/calculators/minimum_wage_calculator.rb`